### PR TITLE
Re-ordering of Collections from the accordion

### PR DESCRIPTION
**Pull request by pmbohm**

### CODE REVIEW

**Comment by anguss00** [comment]
I'll look at this

File: web-app/js/portal/details/NcWmsPanel.js [hunk 1]
```diff
@@ -323,7 +323,8 @@ Portal.details.NcWmsPanel = Ext.extend(Ext.Container, {
         if (this.geoNetworkRecord) {
             this._addDateTimeFilterToLayer();
             this.geoNetworkRecord.updateNcwmsParams(dateRangeStart, dateRangeEnd, geometry);
-            Ext.MsgBus.publish(PORTAL_EVENTS.DATA_COLLECTION_MODIFIED);
+            // todo cant have this for accordion
+            //Ext.MsgBus.publish(PORTAL_EVENTS.DATA_COLLECTION_MODIFIED);
```
Comment (anguss00) [inline web-app/js/portal/details/NcWmsPanel.js]: what happens if this event is activated?

**Comment by anguss00** [comment]
@pmbohm I've found a bug, when you move a layer up or down the accordion the filters are reset

**Comment by pmbohm** [comment]
DONT MERGE - ncWMS subset panel is empty on re-ordering

**Pull request closed without merging**